examples/projections/cyl/cyl_universal_transverse_mercator.py

```diff
@@ -2,19 +2,33 @@
 Universal Transverse Mercator
 =============================
 
-A particular subset of the transverse Mercator is the Universal Transverse
-Mercator (UTM) which was adopted by the US Army for large-scale military maps.
-Here, the globe is divided into 60 zones between 84°S and 84°N, most of which
-are 6° (in longitude) wide. Each of these UTM zones have their unique central
-meridian. Furthermore, each zone is divided into latitude bands but these are
-not needed to specify the projection for most cases.
+A particular subset of the
+:doc:`transverse Merctor </projections/cyl/cyl_transverse_mercator>`
+is the Universal Transverse Mercator (UTM) which was adopted by the US Army
+for large-scale military maps. Here, the globe is divided into 60 zones
+between 84°S and 84°N, most of which are 6° (in longitude) wide.
+Each of these UTM zones have their unique central meridian.
+Furthermore, each zone is divided into latitude bands but these are
+not needed to specify the projection for most cases. See Figure
+:ref:`Universal Transverse Mercator <GMT_utm_zones>` for all zone designations.
+
+.. _GMT_utm_zones:
+
+.. figure:: /_static/GMT_utm_zones.png
```

Comment:
Instead of including the image in the pygmt repository, we can directly include it as an external image, which can keep the pygmt repository small:
```suggestion
.. figure:: https://docs.generic-mapping-tools.org/latest/_images/GMT_utm_zones.png
```

Reply:
Thanks @seisman. I removed the image file "GMT_utm_zones.png" in https://github.com/GenericMappingTools/pygmt/pull/2061/commits/aafcfafc0a89167050f2947ab9eec19db0428d89.